.clang-format

```diff
@@ -40,6 +40,7 @@ BreakBeforeBinaryOperators: NonAssignment
 BreakBeforeTernaryOperators: true
 BreakConstructorInitializersBeforeComma: true
 ColumnLimit:     120
+CommentPragmas: '^(AXIVION|NOLINT)'
```

Comment:
This prevents to merge a line with a lint into a comment from a line above. Unfortunately this does also not break up the line with the lint itself and long lines need to be broken up maually